src/V3AstNodes.cpp

```diff
@@ -829,7 +829,8 @@ AstVar* AstVar::scVarRecurse(AstNode* nodep) {
     return nullptr;
 }
 
-const AstNodeDType* AstNodeDType::skipRefIterp(bool skipConst, bool skipEnum) const VL_MT_STABLE {
+const AstNodeDType* AstNodeDType::skipRefIterp(bool skipConst, bool skipEnum,
+                                               bool assert) const VL_MT_STABLE {
```

Comment:
```suggestion
                                               bool assertOn) const VL_MT_STABLE {
```

std::assert() is a function so this would otherwise hide that.  Update .h similar, thanks.